src/master-playlist-controller.js

```diff
@@ -256,7 +262,22 @@ export class MasterPlaylistController extends videojs.EventTarget {
     // combined audio/video or just video when alternate audio track is selected
     this.mainSegmentLoader_ = new SegmentLoader(segmentLoaderOptions);
     // alternate audio track
+    segmentLoaderOptions.loaderType = 'audio';
     this.audioSegmentLoader_ = new SegmentLoader(segmentLoaderOptions);
+
+    this.decrypter_.onmessage = (event) => {
+      switch (event.data.source) {
+      case 'main':
```

Comment:
vjsstandard linting rules require the cases to be the same indentation of the switch. I too prefer the cases indented